### PR TITLE
STAKAN-1281 transfer bas docs to docusaurus

### PR DESCRIPTION
**Pull request by raproid**

### CODE REVIEW

File: docs/Earn/liquid-staking/eth/eth-liquid-staking.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ id: eth-liquid-staking-introduction
 
 [Ankr Earn](https://www.ankr.com/earn/) offers ETH holders the opportunity to stake ETH and claim two different type of ETH2 Liquid Staking tokens:
 
-![Acala Full Node](../../../../static/img/aethb-aethc-difference.jpeg)
+<img src={require('/img/aethb-aethc-difference.jpeg').default} alt="Diefference between aETHb and aETHc" class="responsive-pic" />
 
 * aETHc is a reward-bearing token, meaning that the fair value of 1 aETHc token vs. ETH increases over time as staking rewards accumulate. When it will be possible to unstake ETH at phase 1.5 of Ethereum 2.0, users will have the option to redeem aETHc to Ankr StakeFi, and unstake ETH with accumulated staking rewards.
```
Comment (ddocs) [inline docs/Earn/liquid-staking/eth/eth-liquid-staking.md:10]: This is an example review in situ e.g. please capitalize or add hyphen. etc.

File: docs/Earn/liquid-staking/eth/eth-liquid-staking.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ id: eth-liquid-staking-introduction
 
 [Ankr Earn](https://www.ankr.com/earn/) offers ETH holders the opportunity to stake ETH and claim two different type of ETH2 Liquid Staking tokens:
```
Comment (ddocs) [inline docs/Earn/liquid-staking/eth/eth-liquid-staking.md:6]: please add 's' to 'type.

**Review by ddocs — COMMENTED** [review]
Great work! Just add the 's'.

File: docs/Earn/liquid-staking/eth/eth-liquid-staking.md (hunk 2 above)
Comment (raproid) [reply docs/Earn/liquid-staking/eth/eth-liquid-staking.md]: done

File: docs/Earn/liquid-staking/eth/eth-liquid-staking.md (hunk 1 above)
Comment (raproid) [reply docs/Earn/liquid-staking/eth/eth-liquid-staking.md:10]: capitalized, hyphened. etc. :D

**Pull request merged**